Sources/StreamChatUI/ChatChannelList/Search/ChatChannelListSearchVC.swift

```diff
@@ -100,8 +100,8 @@ open class ChatChannelListSearchVC: ChatChannelListVC, UISearchResultsUpdating {
 
     // MARK: - State Handling
 
-    override open func controller(_ controller: DataController, didChangeState state: DataController.State) {
-        switch state {
+    override open func handleStateChanges(_ newState: DataController.State) {
```

Comment:
this is a bit of a breaking change though

Reply:
It is a ChatChannelListVC method which is an open method. ChatChannelListSearchVC is a subclass of the ChatChannelListVC, therefore it is not a breaking change - this method is already open.

Reply:
not in the public api, but in the behaviour. This would potentially break clients that have overriden this method here, right?

Reply:
This is not a breaking change, this is called by the other function